### ISSUE TITLE
Browse.path not sent (Go to definition sometimes goes to the declaration instead)

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cmake-tools#1476]
I am working in VSCode Version: 1.49.3 and have this bug, macOS 10.15.4

When I click "Go to definition", sometimes it went to declarations and sometimes it went to the definition.
No error or warning messages.

c_cpp_properties.json

```
{
  "configurations": [
    {
      "name": "vscode_stm32_c_cpp_properties",
      "compilerPath": "/Users/AneoX/compilers/gcc-arm-none-eabi-9-2020-q2-update/bin/arm-none-eabi-gcc",
      "includePath": [
        "${workspaceRoot}/Inc/",
        "${workspaceRoot}/Drivers/cmsis/",
        "${workspaceRoot}/Drivers/STM32F0xx_StdPeriph_Driver/inc/"
        // "${workspaceRoot}/**"
      ],
      "browse": {
        "path": ["${workspaceFolder}"],
        "limitSymbolsToIncludedHeaders": true,
        "databaseFilename": "${workspaceFolder}/.vscode/browse.vc.db"
      },
      "cStandard": "c11",
      "cppStandard": "c++17",
      "intelliSenseMode": "gcc-arm",
      "configurationProvider": "ms-vscode.cmake-tools",
      "defines": ["USE_STDPERIPH_DRIVER", "STM32F042"]
    }
  ],
  "version": 4
}
```

_Originally posted by @aneox in https://github.com/microsoft/vscode-cpptools/issues/3378#issuecomment-703267452_

**Comment by rodrigobrochado** [comment]
Any progress on this. It seems that the extension is overiding the browse.path with an empty list.

**Comment by bobbrow** [comment]
This should have been fixed with #2418.  The fix is now available in the "pre-release" version of CMake Tools.  It will be included in 1.10.  If you would like to try it out and see, please install the pre-release version.
